plugins.d/python.d.plugin

```diff
@@ -505,42 +527,40 @@ def run():
     disabled = []
     configfile = CONFIG_DIR + "python.d.conf"
 
-    interval = INTERVAL
+    update_every = UPDATE_EVERY
     conf = read_config(configfile)
     if conf is not None:
         try:
+            # FIXME: is this right? exit the whole plugin?
```

Comment:
This is intended.
If you set `enabled: no` in `python.d.conf` it disables whole python driver